source/endpoints/devices/Activator.py

```diff
@@ -8,7 +8,7 @@
 activator = namespace.model('Activator', {
     'activatorId': fields.Integer(readOnly=True, required=True, discription='The unique identifier of the action')
     , 'name': fields.String(readOnly=True, required=True, description='The name of the action')
-    , 'type': StateTypeToString(attribute="type", readOnly=True, required=True,
+    , 'type': fields.String(attribute="type", readOnly=True, required=True,
```

Comment:
This was discussed with frank. We wanted to be able to create our own data types. This way it would be easier to communicate a range of values that are viable